endpoints/v2/manifest.py

```diff
@@ -280,14 +311,14 @@ def write_manifest_by_digest(namespace_name, repo_name, manifest_ref):
     )
 
 
-def _parse_manifest():
-    content_type = request.content_type or DOCKER_SCHEMA1_MANIFEST_CONTENT_TYPE
+def _parse_manifest(content_type, request_data):
+    content_type = content_type or DOCKER_SCHEMA1_MANIFEST_CONTENT_TYPE
```

Comment:
Why was this change needed?

Reply:
it will be needed to parse a manifest coming from a request other than `request`, which we'll be doing when storing the manifest.